app/proguard-rules/okhttp3-rules.pro

```diff
@@ -0,0 +1,6 @@
+# OkHttp3
```

Comment:
Yeah, I got the rules from https://github.com/krschultz/android-proguard-snippets/tree/master/libraries which lists a lot of ProGuard rules for various libraries. I didn't check to see if those rules could be cut down though.

Will add a comment to the file with the sources.